CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # main
 
+## 0.12.0
+- BDOrthoVintageExtractor: does not assume the use of VRT but instead relies on specified rgb_file and irc_file
```

Comment:
Maybe to be marked as a breaking change as it expects a different model of the database from the previous versions

Reply:
In this PR nothing different is expected since the DB is not updated yet. I'll indicate the breaking change in the next PR that will update the SQL query.